docs/site/content/posts/04-tce-with-kpack.md

```diff
@@ -0,0 +1,105 @@
+versions of language libraries and operating systems (OSs), ensuring
+that internal compliance standards are satisfied. Further enhancing
+productivity, kpack also detects changes to source code, dependency, or
+OS components and automatically updates containers, using software that
+is approved by the developer's organization.
+
+!["Four Phases of Build Automation: 1. Detect, 2. Restore and Analyze, 3. Build, 4. Export and Cache"](/img/four-phases.png)
+
+Application operators and DevOps teams benefit too. Automated container
+re-builds update images rapidly and efficiently as container components
+are updated, saving time while also enabling operations teams to reduce
+the risks posed by Common Vulnerabilities and Exposures (CVEs).
+Centralized management of container metadata -- including information
+about approved software versions for example -- makes policy enforcement
+easier so that ops teams can make informed image promotion and
+deployment decisions at scale. In addition, as part of the kpack build
+process, bill of materials (BOM) metadata is surfaced for every
+container built, enabling full-stack auditing, tracking, and container
+patching.
+
+Learn more about using kpack, the Buildpacks model for build automation,
+and open source Paketo buildpacks [here](https://tanzu.vmware.com/content/blog/tanzu-community-edition-kpack-kubernetes-container-build-platform).
+
+## Growing developer support in Tanzu Community Edition
+
+With the addition of kpack, Tanzu Community Edition now provides a set
+of open source tooling that supports the core steps of service delivery:
+kpack turns source code into container images; the Harbor container
+registry signs images, scans them for vulnerabilities (using the
+embedded Trivy scanner), and provides secure registry services; Knative
+Serving simplifies deployment and running of services.
+
+!["Building a Path to Production in Tanzu Community Edition now including Open Source software components from Tanzu Application Platform"](/img/path-to-production.png)
+
+Because all of these capabilities are packaged as part of the Tanzu
+Community Edition platform, it is easy for platform operators to provide
+them to their users. In Tanzu Community Edition, kpack, Harbor, and
+Knative, as well as many other platform components, are all provided as
+part of a curated, open source platform distribution that is easy to
+install, can be flexibly configured, and consistently managed.
+
+Reaching this stage of functional support for application development
+and deployment marks an exciting milestone for Tanzu Community Edition.
+And there is more to come. Looking ahead, the Tanzu Community Edition
+project is expected to add supply chain choreography via the
+Cartographer open source project, as well as additional
+developer-focused capabilities that will enable developers to further
+simplify, accelerate, and secure the creation and delivery of cloud
+native workloads.
+
+## Try it out now!
```

Comment:
```suggestion
## Try it out now
```

The linter is unhappy with the exclamation point

Reply:
Yeah. I noted that in a comment above. I don't know if I can change the copy. I'll ask.